ngraph/core/include/ngraph/op/add.hpp

```diff
@@ -106,9 +64,7 @@ namespace ngraph
             };
 
         } // namespace v1
-        NGRAPH_SUPPRESS_DEPRECATED_START
-        using v0::Add;
-        NGRAPH_SUPPRESS_DEPRECATED_END
+        using v1::Add;
```

Comment:
Please do not use using. it was introduced for nGraph backward compatibility.

Reply:
Fixed